frontend/src/components/App.js

```diff
@@ -45,6 +47,28 @@ export default function App() {
           element={<div>This site was made with Solid</div>}
         />
       </Routes>
+      </div>
+
+      {/* Footer div */}
```

Comment:
Can you pull the components out into separate modules like `Footer.js` and `Header.js` ?

Reply:
Also, since the footer has About link, it can be removed from the header.

Reply:
okay